docker/compose/suzuka-full-node/docker-compose.ledger-should-progress.yml

```diff
@@ -0,0 +1,70 @@
+services:
+
+  # This service checks if the ledger version has increased every LEDGER_INCREASE_PERIOD_SECONDS
+  ledger-should-progress:
+    image: alpine:latest
+    container_name: ledger-should-progress
+    command: >
+      sh -c '
```

Comment:
The script is a copy of `ledger-should-progress.sh`, leave just one?

Reply:
#717